games-strategy/freeciv/freeciv-3.0.10.ebuild

```diff
@@ -0,0 +1,193 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
Remember to update these on merge... run **pkgcheck scan --commits** before pushing.